extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -531,9 +655,6 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 				TelemetryAction.ServerAssessment,
 				{
 					'sessionId': this._sessionId,
-					'tenantId': this._azureAccount.properties.tenants[0].id,
```

Comment:
Even though we're removing the tenantId, subscriptionId, resourceGroup during generateAssessment(), we still need this information. Can we add these data points on the targetSelectionPage?

We can send the telemetry onPageLeave once the user selects these values.
https://github.com/microsoft/azuredatastudio/blob/854e90b772537abc677e7e54d01124c8e4d18c2d/extensions/sql-migration/src/wizard/targetSelectionPage.ts#L131

Reply:
You mean just the tenantId,subid, rg? or entire assessment telemetry.

Reply:
Done